website/src/pages/api/chat/index.ts

```diff
@@ -0,0 +1,12 @@
+import { post } from "src/lib/api";
+import { withoutRole } from "src/lib/auth";
+
+// TODO: move to env
+export const INFERENCE_URL = "http://localhost:8000";
+
+const handler = withoutRole("banned", async (req, res, token) => {
```

Comment:
Do we want to try calling the inference service directly from the client side without forwarding it through the next api routes?

Reply:
I only proxy to make sure the user is authenticated & authorized, not sure what other alternatives we have.

Reply:
that makes sense. i  need to get unlazy and setup JWT handling on the inference server.  Then we can rely on the JWT contents for this verification.